app/angular/src/client/preview/angular/NgModulesAnalyzer.ts

```diff
@@ -0,0 +1,59 @@
+import { NgModule } from '@angular/core';
+
+/**
+ * Avoid component redeclaration
+ *
+ * Checks recursively if the component has already been declared in all import Module
+ */
+export const getExistenceOfComponentInModules = (
```

Comment:
`isComponentAlreadyDeclaredInModules`?

Reply:
haha I had kept the original name of this function. Good suggestion I change it ;)